manifest.json

```diff
@@ -494,13 +496,14 @@
               "size": "small",
               "mode": "exhaustive search",
               "features": [
+                "state constraint",
```

Comment:
State constraints and liveness generally do not play well with each other, was this taken into account?

Reply:
Yes, and I verified that it will be broken if it's not fulfilled (by putting in a temporary error in the model).